docs/UserGuide.md

```diff
@@ -759,7 +759,7 @@ You just purchased a bottle of conditioner for your client's hair treatment toda
 You can follow the steps below to add the expense to GrAB3.
 
 Steps :
-1. Type `addexp d/conditioner f/f v/15.00 dt/28-10-2020 t/hair supplies` in to _Command Box_.
+1. Type `addexp d/conditioner f/f v/15.00 dt/28-10-2020 t/hairsupplies` in to _Command Box_.
```

Comment:
Cannot handle white space?

Reply:
I'm currently using the Tag Class provided by AB3 which does not handle white space, will handle it subsequently if it is consistent with the other features using the same Tag Class.